templates/App.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+  import Button from './Button.svelte';
+  import Svg from './Svg.svelte';
+</script>
+
+<h3>Buttons</h3>
+
+<Button>Base button</Button>
+<Button type="solid">Button solid</Button>
+<Button type="link">Button link</Button>
+
+<code>
+  {`<Button>Base button</Button>`}<br>
+  {'<Button type="solid">Solid button</Button>'}<br>
+  {'<Button type="link">Button link</Button>'}
+</code>
+
+<h3>Styles</h3>
+
+<Button style="primary">Button primary</Button>
+<Button style="secondary">Button secondary</Button>
+<Button style="accept">Button accept</Button>
+<Button style="cancel">Button cancel</Button>
+
+<code>
+  {'<Button style="primary">Button primary</Button>'}<br>
+  {'<Button style="secondary">Button secondary</Button>'}<br>
+  {'<Button style="accept">Button accept</Button>'}<br>
+  {'<Button style="cancel">Button cancel</Button>'}
+</code>
+
+<h3>Icons</h3>
+
+<Button style="primary">
```

Comment:
Det ser da pænt ud! Ligger <Button> ikke lidt tæt på <button>? Skal vi overveje et prefix?

Reply:
Tænkte godt selv på det. men hvad skulle fordelen være når det som sådan ikke er tænkt som noget der skal deles med verden?

Reply:
tanken er vel at alle `<button>` skal være `<Button>`

Reply:
Det var nu mere i forhold til, når vi skulle bruge det. Hvis man lige glemmer shift, så har man et "andet" komponent - også ift. læsbarhed. HTML er heller ikke case-sensitive, så komponentet virker potentielt som `<button>`, hvis man ikke har inkluderet vores definition = mulighed for fejl.

Reply:
det er der bestemt en pointe i - hvis vi prefixede vores designsystem komponenter, så vil det også give bedre mulighed for at have projekt specifikke komponenter eller ... importerer nogle andres komponenter